gulp/watch.js

```diff
@@ -114,7 +114,7 @@ module.exports = function (gulp, gutil) {
   });
 
   gulp.task('watch-res-sass', function (cb) {
-    watch()(watchSrc(['res/**/' + server.localSkinCompileVersion() + '/**/*.{scss,properties}']), watchOpts, function (file) {
+      watch()(watchSrc(['res/feature/responsivepeak/' + server.localSkinCompileVersion() + '/**/*.{scss,properties}', 'res/**/!(responsivepeak)/**/*.{scss,properties}']), watchOpts, function (file) {
```

Comment:
I think the indentation should only be 2 spaces here.

Also, lines should not be longer than 120 characters, please make it two lines.